core/domain/draft_upgrade_services.py

```diff
@@ -134,89 +134,107 @@ def _convert_html_in_draft_change_list(
             # The change object has the key 'new_value' only if the
             # cmd is 'CMD_EDIT_STATE_PROPERTY' or
             # 'CMD_EDIT_EXPLORATION_PROPERTY'.
-            new_value = change.new_value
-            if change.property_name == exp_domain.STATE_PROPERTY_CONTENT:
-                # Ruling out the possibility of any other type for MyPy
-                # type checking, because here 'new_value' is a value of
-                # attribute that was dynamically defined on change_domain
-                # and that attribute has str type. Thus to convert the
-                # type, we used assert here.
-                assert isinstance(new_value, dict)
-                new_value['html'] = conversion_fn(new_value['html'])
-            elif (change.property_name ==
+            # Here we use cast because we are narrowing down the type from
+            # ExplorationChange to a specific change command.
+            edit_state_property_cmd = cast(
+                exp_domain.EditExplorationStatePropertyCmd,
+                change
+            )
+            new_value = edit_state_property_cmd.new_value
+            if (
+                edit_state_property_cmd.property_name ==
+                exp_domain.STATE_PROPERTY_CONTENT
+            ):
+                # Here we use cast because in this 'if clause' we are
+                # updating the content of a State which can only be of
+                # type SubtitledHtmlDict. So, to rule out all other
+                # property types for MyPy type checking, we used cast here.
+                state_content = cast(state_domain.SubtitledHtmlDict, new_value)
+                state_content['html'] = conversion_fn(state_content['html'])
+            elif (edit_state_property_cmd.property_name ==
                   exp_domain.STATE_PROPERTY_INTERACTION_CUST_ARGS):
-                # Only customization args with the key 'choices' have HTML
-                # content in them.
-                # Ruling out the possibility of any other type for MyPy type
-                # checking because here 'new_value' is a value of attribute
-                # that was dynamically defined on change_domain and that
-                # attribute has str type. Thus to convert the type, we used
-                # assert here.
-                assert isinstance(new_value, dict)
-                if 'choices' in new_value.keys():
+                # Here we use cast because in this 'elif clause' we are
+                # updating the interaction's customization args of a State
+                # which can only be of type Dict[str, Dict[str, Any]]. So,
```

Comment:
This seems wrong, it should be a union of the different customization arg dicts that we talked about for the other problem you are solving. Don't use Any.

Reply:
As per the plan, a new PR is going to be raised for the customization args. So, currently, these interaction_customization_args are pointed to `CustomizationArgsDictType`
https://github.com/oppia/oppia/blob/37ff4591fc7e57f273e54ac5a0226b8561086dbf/core/domain/state_domain.py#L68
 but once the PR is raised, the union of all  interaction_customization_args are named as `CustomizationArgsDictType` -- so that these customization args points to union of all  interaction_customization_args automatically.